modules/reference/pages/k-crd.adoc

```diff
@@ -8,42 +8,154 @@
 Package v1alpha2 defines the v1alpha2 schema for the Redpanda API. It is part of an evolving API architecture, representing an initial stage that may be subject to change based on user feedback and further development.
 
 .Resource Types
```

Comment:
only the items listed here are resources 😄 they are the Kubernetes objects you can create and manage in the cluster. The other sections are parts of those objects that you can configure. It's confusing because of the heading levels, but the resources can share the configuration parts so there's no real hierarchy.